src/commands/dxb/profile/passwordpolicies/fix.ts

```diff
@@ -29,26 +27,48 @@ export default class PasswordPoliciesMerge extends SfdxCommand {
     protected orgname:string;
 
     public async run() {
-        let profilename = this.flags.profilename;
-        let sourcepath = this.flags.sourcepath;
-        this.orgname = this.org.getUsername();
-        if (profilename) {
-            try {
-                //do something
-            } catch (err) {
-                console.log(`Could not convert ${profilename}`);
+        const sourcepath = this.flags.sourcepath;
+        try {
+            // check if sourcepath exists and continue
+            fs.accessSync(sourcepath);
+            // retrieve all password policies from target org, these have a different timestamp appended to the file name than the source files
+            fs.mkdirSync('targetOrgPolicies/main/default', { recursive: true });
+            const targetOrgPolicies: { filePath: string, fullName: string }[] = this.getAllPasswordPolicies();
+
+            // get the file names for the source files from the source directory
+            let sourceFiles:string[] = fs.readdirSync(sourcepath);
+            if (sourceFiles.length === 0) {
+                throw new Error(`No source files were found in ${sourcepath}`);
+            }
+            for (const file of sourceFiles) {
+                // find the profile password policy from the target org that starts with the same profile name as the source file
+                const targetOrgPolicyToReplace:string = (
```

Comment:
What if a password policies doesn't exist in target yet ?

Reply:
@davidbrowaeys hence the exception where I return an object { filePath: `targetOrgPolicies/profilePasswordPolicies/${file}`}. If the password policy for that profile does not exist in target, the source file should be copied over.